packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -3112,6 +3123,12 @@ export type OnRenderAvatarCallback = (
 userId?: string, options?: CustomAvatarOptions,
 defaultOnRender?: (props: CustomAvatarOptions) => JSX.Element) => JSX.Element;
 
+// @public (undocumented)
```

Comment:
Do we expose lazy loading as public or beta? I see some of them marked as public but other marked as beta